spec/controllers/profile_controller_spec.rb

```diff
@@ -16,5 +16,15 @@
       get :show, params: { user_id: @user }
       expect(@user.profile).to be
     end
+
+    it 'should use the profile presenter for the profile' do
```

Comment:
Could you write this as `uses the profile presenter for the profile`? I know you were just copying the `should`s already in here since you didn't  use them in your next spec file, but I think I'd rather cope with the inconsistency within the file than add another unnecessary `should` 😉